Tests/Unit/Module/STIG.PowerStigXml.tests.ps1

```diff
@@ -0,0 +1,42 @@
+#region Header
+. $PSScriptRoot\.tests.header.ps1
+#endregion
+
+$xccdfs = (Get-ChildItem -Path $script:moduleRoot\StigData\Archive -Include *xccdf.xml -Recurse | Where-Object -Property Name -Match "Server_2019_MS|IIS_10-0_Server")[1,3]
+foreach ($xccdf in $xccdfs)
+{
+    Describe "ConvertTo-PowerStigXml $($xccdf.name)" {
+
+        It 'Should return an 2 XML' {
+            ConvertTo-PowerStigXml -Path $xccdf.FullName -Destination $TestDrive -CreateOrgSettingsFile -RuleIdFilter $randomId
+            $converted = Get-ChildItem $testdrive
```

Comment:
missing named -Path parameter for Get-ChildItem.

Reply:
Updated